frontend/src/components/elements/Video/Video.tsx

```diff
@@ -42,7 +42,40 @@ class Video extends React.PureComponent<Props> {
   }
 
   public render(): React.ReactNode {
+    /* Element may contain "url" or "data" property. */
+
     const { element, width } = this.props
+
+    if (element.get("url")) {
+      /* is this a YouTube link? if so we need a fancier tag. 
+         NOTE: This part assumes the URL is already an "embed" link.
+      */
+      if (element.get("url").includes("youtube.com")) {
+        const height = width * 0.75
+        const wid = width
```

Comment:
wid → width

Reply:
I couldn't get this change to work, it complained of a missing comma...

Reply:
Actually the problem is that `width` already exists. So you don't need the line above — just use `width` directly below